docs/api.rst

```diff
@@ -6,3 +6,40 @@
 
 .. automodule:: adafruit_midi
```

Comment:
Change this to `.. automodule:: adafruit_midi.adafruit_midi` to match the module name change in `adafruit_midi/__init__.py`.

Reply:
I get an error if I do that.

```
$ ( cd docs && sphinx-build -E -W -b html . _build/html )
Running Sphinx v1.8.5
loading intersphinx inventory from https://docs.python.org/3.4/objects.inv...
loading intersphinx inventory from https://circuitpython.readthedocs.io/en/latest/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 3 source files that are out of date
updating environment: 3 added, 0 changed, 0 removed
reading sources... [100%] index

Warning, treated as error:
autodoc: failed to import module 'adafruit_midi' from module 'adafruit_midi'; the following exception was raised:
No module named 'adafruit_midi.adafruit_midi'
```

My version of `sphinx-build` at home is ok with it as is. That's odd because the Travis one I presume is still choking on some part of this.

Reply:
Did you change the module name in `adafruit_midi/__init__.py` to `adafruit_midi.adafruit_midi`? If not, then I guess you could try without changing either of them. But I'm not sure autodoc will find it...

Reply:
Version of `spinhx-build` I'm using works ok with just `adafruit_midi` in docs/api.rst

The version Travis uses is failing but that's something obscure to do with the mocking on the usb_device objects. I didn't introduce those objects, they were already there, puzzling!

Reply:
I have narrowed down the sphinx failure. This should stay as it is.